src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -298,6 +298,9 @@
         - name: --enable-pod-identity
           type: bool
           short-summary: (PREVIEW) Enable pod identity addon.
+        - name: --enable-pod-identity-with-kubenet
```

Comment:
is `-with-kubenet` required here? kubenet should be set with --network-plugin=kubenet

Reply:
this flag means we want to enable pod identity addon for cluster using kubenet. It will not set the cluster plugin to kubenet. When we try to turn on the plugin, it works like:

- check if the cluster is using kubenet network plugin?
  * no: we are good
  * yes: do we set the `--enable-pod-identity-with-kubenet` consent?
    * no: raise error
    * yes: user consent set, we can enable this addon

Reply:
since Pod identity works for both azure cni and kubenet, why not reusing `--enable-pod-identity`?

Reply:
this is because there is a known security issue for enabling this addon in kubenet cluster: https://azure.github.io/aad-pod-identity/docs/configure/aad_pod_identity_on_kubenet/ Therefore, we add this consent flag to make sure the user acknowledges the rick before using it. We will put the details in the azure doc. cc @miwithro for the doc

Reply:
OK, thanks for the clarification